Source/Core/DolphinQt/NetPlay/NetPlaySetupDialog.cpp

```diff
@@ -110,17 +110,20 @@ void NetPlaySetupDialog::CreateMainLayout()
   connection_layout->addWidget(m_connect_port_label, 0, 2);
   connection_layout->addWidget(m_connect_port_box, 0, 3);
   connection_layout->addWidget(
-      new QLabel(
-          tr("ALERT:\n\n"
-             "All players must use the same Dolphin version.\n"
-             "If enabled, SD cards must be identical between players.\n"
-             "If DSP LLE is used, DSP ROMs must be identical between players.\n"
-             "If a game is hanging on boot, it may not support Dual Core Netplay."
-             " Disable Dual Core.\n"
-             "If connecting directly, the host must have the chosen UDP port open/forwarded!\n"
-             "\n"
-             "Wii Remote support in netplay is experimental and may not work correctly.\n"
-             "Use at your own risk.\n")),
+      new QLabel(tr(
+          "ALERT:\n\n"
+          "Netplay will use the input configuration from controller port 1 first,\n"
+          "regardless of the port you are assigned during the netplay session.\n"
+          "Wii Remotes, on the other hand, use the configuration from the assigned netplay slot.\n"
+          "All players must use the same Dolphin version.\n"
+          "If enabled, SD cards must be identical between players.\n"
+          "If DSP LLE is used, DSP ROMs must be identical between players.\n"
+          "If a game is hanging on boot, it may not support Dual Core Netplay."
+          " Disable Dual Core.\n"
+          "If connecting directly, the host must have the chosen UDP port open/forwarded!\n"
+          "\n"
+          "Wii Remote support in netplay is experimental and may not work correctly.\n"
+          "Use at your own risk.\n")),
```

Comment:
Might as well fix all the case inconsistencies while we're messing with this. :)

I also cleaned up the phrasing a little bit, and changed some spacing.

```suggestion
          "ALERT:\n\n"
          "GameCube Controllers will use the input configuration starting from controller port 1,"
          " regardless of the port you are assigned during the NetPlay session.\n"
          "Wii Remotes, on the other hand, use the configuration corresponding to your"
          " assigned port in the NetPlay session.\n"
          "\n"
          "All players must use the same Dolphin version.\n"
          "If enabled, SD cards must be identical between players.\n"
          "If DSP LLE is used, DSP ROMs must be identical between players.\n"
          "If a game is hanging on boot, it may not support Dual Core NetPlay."
          " Disable Dual Core.\n"
          "If connecting directly, the host must have the chosen UDP port open/forwarded!\n"
          "\n"
          "Wii Remote support in NetPlay is experimental and may not work correctly.\n"
          "Use at your own risk.\n")),
```